content/posts/this-month-in-hax/2024-07.md

```diff
@@ -0,0 +1,70 @@
+* #799: [fix(bounded-ints): zero and one were inverted](https://github.com/hacspec/hax/pull/799)
+* #797: [Fix potentially looping predicate_id.](https://github.com/hacspec/hax/pull/797)
+* #796: [Add nicer error messages when rustc emits an error in detect_forking.](https://github.com/hacspec/hax/pull/796)
+* #795: [fix: precondition constraints should be of the shape `true => pred`](https://github.com/hacspec/hax/pull/795)
+* #794: [Resugar asserts](https://github.com/hacspec/hax/pull/794)
+* #792: [feat(frontend): Add option to control output directory](https://github.com/hacspec/hax/pull/792)
+* #790: [Allow `ensures` and `requires` on traits](https://github.com/hacspec/hax/pull/790)
+* #788: [Fix #296](https://github.com/hacspec/hax/pull/788)
+* #779: [Support let-else pattern.](https://github.com/hacspec/hax/pull/779)
+* #777: [Small typo fixes for hax book](https://github.com/hacspec/hax/pull/777)
+* #776: [feat(book): fix checkboxes](https://github.com/hacspec/hax/pull/776)
+* #769: [Extensions to the proof-libs for libcrux-ml-kem](https://github.com/hacspec/hax/pull/769)
+* #765: [fix(engine/lib): import associated item projection on generic bounds](https://github.com/hacspec/hax/pull/765)
+* #764: [feat(hax-lib): allow requires & ensures on `&mut` inputs functions](https://github.com/hacspec/hax/pull/764)
+* #763: [fix(hax-lib): refinements: various fixes](https://github.com/hacspec/hax/pull/763)
+* #762: [fix(hax-lib): `use super::*` in refinement expansion](https://github.com/hacspec/hax/pull/762)
+* #759: [fix(frontend): crashes on fn ptr](https://github.com/hacspec/hax/pull/759)
+* #758: [fix color printing in util script](https://github.com/hacspec/hax/pull/758)
+* #756: [fix(frontend): kill `crate_type` in `HaxMeta`](https://github.com/hacspec/hax/pull/756)
+* #753: [ exporter: make it need nightly only when feature `rustc` is on](https://github.com/hacspec/hax/pull/753)
+* #751: [Engine: propagate trait generics arguments](https://github.com/hacspec/hax/pull/751)
+* #750: [feat(exporter): thir: call: separate trait VS method generic args](https://github.com/hacspec/hax/pull/750)
+* #744: [doc(engine): ppx_functor_application](https://github.com/hacspec/hax/pull/744)
+* #743: [Refactor of the frontend](https://github.com/hacspec/hax/pull/743)
+* #730: [fix(frontend): make `path_to` breadth-first](https://github.com/hacspec/hax/pull/730)
+* #729: [Move book from `hacspec/book` to `hacspec/hax`](https://github.com/hacspec/hax/pull/729)
+* #727: [fix(engine): `Concrete_ident_generated`: `name` -> `t`, derive more](https://github.com/hacspec/hax/pull/727)
+* #698: [Generate visitors automatically](https://github.com/hacspec/hax/pull/698)
+
+# Contributors
+[paulmure](https://github.com/paulmure)
+[maximebuyse](https://github.com/maximebuyse)
+[karthikbhargavan](https://github.com/karthikbhargavan)
+[W95Psp](https://github.com/W95Psp)
```

Comment:
A list looks a little nicer here I think.

```suggestion
- [paulmure](https://github.com/paulmure)
- [maximebuyse](https://github.com/maximebuyse)
- [karthikbhargavan](https://github.com/karthikbhargavan)
- [W95Psp](https://github.com/W95Psp)
```

Reply:
Oh, right, I meant to do so, I forgot, thanks!